MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffFittingPresenter.cpp

```diff
@@ -709,6 +722,121 @@ std::string EnggDiffFittingPresenter::functionStrFactory(
   return functionStr;
 }
 
+void EnggDiffFittingPresenter::browsePeaksToFit() {
+  try {
+    QString prevPath = QString::fromStdString(m_view->focusingDir());
```

Comment:
`QString` should be avoided in the presenter. This comes from code that was in the view and was using `QString` for the Qt functions, but here there is no need to convert the `std::string` to a `QString`.